### ISSUE TITLE
[T3] Problema con router

### ISSUE THREAD

**Issue by luzmagurzua** [IIC2513/Syllabus-S1-2024-1#155]
Hola! En el archivo app.js del backend, no debería ser require('./routes) ? (linea 4)

![Captura de Pantalla 2024-04-22 a la(s) 14 18 17](https://github.com/IIC2513/Syllabus-S1/assets/80796978/cbe3e377-86a2-4258-90a4-dce4728da838)

**Comment by renatasantander** [comment]
No, en app.js se importa el archivo routes.js, ya que este último es el enrutador de la aplicación. En el directorio 'routes' se encuentran los archivos users.js y entries.js donde deben definir los endpoints correspondientes. Luego, en routes.js deben importar los módulos de rutas desde el directorio 'routes' con `require` y registrar esas rutas en el enrutador (`/users` y `/entries`).
